src/ui/screens/Council/CouncilScreen.tsx

```diff
@@ -283,6 +304,94 @@ function CouncilVoteModal({visible, setVisible, candidates, moduleElection}: Cou
   );
 }
 
+function SubmitCandidancyModel({visible, setVisible, moduleElection}: SubmitCandidancyProps) {
+  const [account, setAccount] = React.useState<Account>();
+  const startTx = useApiTx();
+  const {api} = useApi();
+  const balance = useFormatBalance();
+  const formattedBalance = balance.formatBalance(moduleElection.candidacyBond);
+  const {data: council} = useCouncil();
+  const {colors} = useTheme();
+  const onSubmitCandidacy = () => {
+    console.log(moduleElection.module);
+    if (account && api && moduleElection.module) {
+      startTx({
+        address: account.address,
+        txMethod: `${moduleElection.module}.submitCandidacy`,
+        params: [
+          api.tx[moduleElection.module]?.submitCandidacy?.meta.args.length === 1 ? [council?.candidates.length] : [],
+        ],
+      });
+      reset();
+    }
+  };
+
+  const reset = () => {
+    setAccount(undefined);
+    setVisible(false);
+  };
+
+  const accountPopableProps = {
+    content: 'Select the account you wish to submit for candidacy.',
+    children: null,
+    backgroundColor: colors.accent,
+  };
+
+  const candidancyBondPopableProps = {
+    content: 'Select the account you wish to submit for candidacy.',
+    children: null,
+    backgroundColor: colors.accent,
+  };
+
+  return (
+    <Modal visible={visible} onDismiss={reset}>
+      <View style={styles.centerAlign}>
+        <Subheading>{`Submit Council Candidancy`}</Subheading>
+      </View>
+      <Padder scale={1} />
+      <Caption>
+        {`Candidate account:`}
+        <Popover
+          popableProps={accountPopableProps}
+          popableContent={
+            <>
+              <View style={styles.paddingTop}>
+                <Icon name={'questionmark'} size={19} />
+              </View>
+            </>
+          }
+        />
+      </Caption>
```

Comment:
I think we should have a generic `<InputLabel />` component roughly like following 

```
      <View style={globalStyles.rowAlignCenter}>
        <Caption>{`Candidate account:`}</Caption>
        {helperText && <Popover popableProps={accountPopableProps} popableContent={<Icon name={`help-circle`} color={colorGray} size={20} />} />}
      </View>
```

Which accepts props `label: String` and `helperText?: String` and construct the necessary popover props from there.

It can also live in `ui/library`